src/components/view.js

```diff
@@ -48,6 +48,12 @@ export class View {
     // Do nothing by default. Override if needed.
   }
 
+  /*
+   * Accept the result.
+   * @return {!Promise}
+   */
+  acceptResult() {}
```

Comment:
I think we need a `whenComplete` as `!Promise` here instead. Otherwise I don't see how we can reconcile types give that acceptResult is `!Promise<ActivityResult>` or what not.

Reply:
Done.